fix(ui): improve TokenValue and liquidity dialog layouts

valeryV99 opened the pull request:
https://www.notion.so/254362757c98816eaa18db14ba3bcb0d?v=254362757c9881b38afd000c1ae9b764&p=25d362757c9880ba81dadaf53e08a9ca&pm=s
- Adjusted `TokenValue` component to improve symbol rendering with ellipsis and tooltip for overflow cases.
- Refactored liquidity dialogs to enhance alignment and layout consistency by replacing `hstack` with `vstack`.
- Removed redundant code for token logos and symbols in liquidity dialogs.

valeryV99 committed:
fix(ui): improve TokenValue and liquidity dialog layouts

- Adjusted `TokenValue` component to improve symbol rendering with ellipsis and tooltip for overflow cases.
- Refactored liquidity dialogs to enhance alignment and layout consistency by replacing `hstack` with `vstack`.
- Removed redundant code for token logos and symbols in liquidity dialogs.

valeryV99 committed:
feat(chart): enhance configuration and adjust visible range logic

- Added `scaleMargins` configuration for charts to improve visual spacing.
- Updated visible range logic to extend `from` value by -14 for better time scale adjustments.

opcode0x commented:
<img width="520" height="568" alt="image" src="https://github.com/user-attachments/assets/fb23f078-1d02-429e-94ac-2ddda01d373f" />

At the moment we display Symbol which is equal for any other rune, should we use rune name instead?

opcode0x commented on apps/web/src/widgets/chart/logic/drawChart.ts:
```diff
@@ -90,8 +90,8 @@ export const drawChart = ({
     const visibleRange = chart.timeScale().getVisibleLogicalRange();
```
I think this is a part of a separate PR

valeryV99 committed:
fixed wrong changes

valeryV99 committed:
fix(ui): enhance TokenValue and liquidity dialog usability

- Updated `TokenValue` component to use token name in tooltip and display instead of symbol.
- Improved layout for liquidity dialogs by adding better ellipsis handling and alignment for pool token symbols.
